manim/mobject/graphing/coordinate_systems.py

```diff
@@ -1836,12 +1836,19 @@ def construct(self):
 
         return T_label_group
 
-    def __matmul__(self, coord: Point3D | Mobject):
+    def __matmul__(self, coord: Iterable[float] | Mobject):
         if isinstance(coord, Mobject):
             coord = coord.get_center()
         return self.coords_to_point(*coord)
 
     def __rmatmul__(self, point: Point3D):
```

Comment:
Maybe the return type should be something like this?
```suggestion
    def __rmatmul__(self, point: Point3D) -> Point2D | Point3D | complex:
```